4902.feature.rst

```diff
@@ -0,0 +1,7 @@
+Added a new configuration parameter, ``ranking_length`` to the ``EmbeddingPolicy``, ``EmbeddingIntentClassifier``,
+and ``ResponseSelector`` classes. When used in combination with ``softmax`` loss type, confidence levels will
+be normalized over the ``ranking_length`` number of top results.
```

Comment:
do we need the explanation in changelog?

Reply:
Will link to explanation in the migration guidw